apps/condo/bin/migrate-user-import-id-data-to-user-external-identity.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * Migrate User.importId/User.importRemoteSystem fields data to separate entity: UserExternalIdentity
+ *
+ * Usage:
+ *      yarn workspace @app/condo node bin/migrate-user-import-id-data-to-user-external-identity
+ */
+
+const path = require('path')
+const dayjs = require('dayjs')
+
+const { GraphQLApp } = require('@keystonejs/app-graphql')
+const { getSchemaCtx } = require('@open-condo/keystone/schema')
+
+const { User, UserExternalIdentity } = require('@condo/domains/user/utils/serverSchema')
+const { IDP_TYPES } = require('@condo/domains/user/constants/common')
+
+
+// init utils & common vars
+const dv = 1
+const sender = { dv, fingerprint: 'migrate-user-importId-to-externalIdentity' }
```

Comment:
```suggestion
const sender = { dv, fingerprint: 'migrate-user-import-data-to-external-identity' }
```

Reply:
fingerprint has max length 42 characters. I'll keep current one for now